tools/pylint/pylintrc

```diff
@@ -297,7 +297,7 @@ ignore-comments=yes
 ignore-docstrings=yes
 
 # Ignore imports when computing similarities.
-ignore-imports=no
+ignore-imports=yes
```

Comment:
What is the effect of this change?

Reply:
Maybe we could rewrite the code to avoid duplicates, if it is not too ugly.

Reply:
Renaming the `get_arguments` function in one of the files is enough.